apps/react-18-tests-v9/jest.config.js

```diff
@@ -6,14 +6,14 @@
 module.exports = {
   displayName: 'react-18-tests-v9',
   preset: '../../jest.preset.js',
-  globals: {
-    'ts-jest': {
-      tsconfig: '<rootDir>/tsconfig.spec.json',
-      isolatedModules: true,
-    },
-  },
   transform: {
-    '^.+\\.tsx?$': 'ts-jest',
+    '^.+\\.tsx?$': [
+      'ts-jest',
```

Comment:
would be nice to get rid of ts-jest at some point, we have type checking for our test files already. Using babel might also make our tests run a lot faster